govcd/vapp_test.go

```diff
@@ -1490,3 +1490,59 @@ func createVappForTest(vcd *TestVCD, vappName string) (*VApp, error) {
 
 	return vapp, nil
 }
+
+// Test_AddNewVMFromMultiVmTemplate creates VM from OVA holding a few VMs
+func (vcd *TestVCD) Test_AddNewVMFromMultiVmTemplate(check *C) {
+
+	if vcd.skipVappTests {
+		check.Skip("Skipping test because vapp was not successfully created at setup")
+	}
+
+	// Find VApp
+	if vcd.vapp.VApp == nil {
+		check.Skip("skipping test because no vApp is found")
+	}
+
+	// Populate Catalog
+	catalog, err := vcd.org.GetCatalogByName(vcd.config.VCD.Catalog.Name, false)
+	check.Assert(err, IsNil)
+	check.Assert(catalog, NotNil)
+
+	itemName := check.TestName()
+	uploadTask, err := catalog.UploadOvf(vcd.config.OVA.OvaMultiVmPath, itemName, "upload from test", 1024)
```

Comment:
We need to check that `OvaMultiVmPath` was set in the configuration.

Moreover, if the user has already a multi-VM template in a catalog, we want to allow the test to run without uploading a new template.

Reply:
Added check and two additional configuration parameters